server/src/configs/server.js

```diff
@@ -0,0 +1,40 @@
+import express from 'express';
+
+const app = () => {
+    let application;
+    const server = express();
+
+    const create = () => {
+        server.set('port', 3030);
```

Comment:
use env variable for port.

Reply:
> What are these .keep files?

These are there just to preserve the folder structure so others can work accordingly. They will be deleted once we have files in the folders.